yarn.lock

```diff
@@ -4,38 +4,38 @@
 
 "@4chain-ag/react-configuration@^1.0.4":
   version "1.0.4"
-  resolved "https://registry.yarnpkg.com/@4chain-ag/react-configuration/-/react-configuration-1.0.4.tgz#4fb9dc952fbd0af88d22731ff3a6adbbab2eec7f"
+  resolved "https://registry.npmjs.org/@4chain-ag/react-configuration/-/react-configuration-1.0.4.tgz"
```

Comment:
Was it necessary to change from `yarnpkg` to `npmjs` ?

Reply:
Reverted